src/authorizations/components/redesigned/GenerateTokenDropdown.tsx

```diff
@@ -26,8 +31,13 @@ const GenerateTokenDropdown: FC<ReduxProps & GenerateTokenProps> = ({
     onShowOverlay('add-master-token', null, onDismissOverlay)
   }
 
-  const handleCustomApi = () => {
-    onShowOverlay('add-custom-token', null, onDismissOverlay)
+  const handleCustomApi = async () => {
+    try {
+      await getAllResources()
+      onShowOverlay('add-custom-token', null, onDismissOverlay)
+    } catch (e) {
```

Comment:
minor nit: I don't think this [`(e)` is necessary](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#the_exception_identifier) if it's not being used, but it's a minor thing. It's more just to make it slightly easier to read / understand.

Reply:
Okay, will remove it in another pr